name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/BaaderBankPDFExtractor.java

```diff
@@ -325,6 +331,12 @@ private void addAdvanceTaxTransaction()
                     t.setAmount(asAmount(v.get("amount")));
                     t.setDateTime(asDate(v.get("date")));
                 })
+                
+                // Ex-Dividenden-Tag
```

Comment:
// Ex-Dividenden-Tag
not the same for regEx
`^Ex-Tag: (?<exDate>\\d+.\\d+.\\d{4})$`

Reply:
changed